src/Symfony/Component/HttpClient/Tests/MockHttpClientTest.php

```diff
@@ -20,6 +20,9 @@
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 use Symfony\Contracts\HttpClient\ResponseInterface;
 
+/**
+ * @group time-sensitive
```

Comment:
same here: sleeps run in a separate webserver process
also: this could work better with a `<element key="time-sensitive"><string>Symfony\Contracts\HttpClient\Test</string></element>` entry in `phpunit.xml.dist`? See eg in HttpKernel's

Reply:
reverted too